docs/pages/components/ActionMenu.mdx

```diff
@@ -0,0 +1,424 @@
+    if (colors.includes(key)) {
+      setColors(colors.filter(value => value !== key))
+    } else {
+      setColors([...colors, key])
+    }
+  }
+
+  return (
+    <ActionMenu
+      trigger={
+        <Button variant="secondary">
+          Setup
+          <Icon type="chevron-down" />
+        </Button>
+      }
+    >
+      <ActionList selectionType="multiple">
+        <ActionListGroup 
+          selectionType="single" 
+          label="Difficulty Level"
+          onAction={handleDifficultyAction}
+        >
+          <ActionListItem selected={difficulty === 'Standard'}>Standard</ActionListItem>
+          <ActionListItem selected={difficulty === 'Normal'}>Normal</ActionListItem>
+          <ActionListItem selected={difficulty === 'Expert'}>Expert</ActionListItem>
+        </ActionListGroup>
+        <ActionListGroup 
+          label="Allowed Fruit"
+          onAction={handleColorsAction}
+        >
+          <ActionListItem selected={colors.includes('Apple')}>Apple</ActionListItem>
+          <ActionListItem selected={colors.includes('Banana')}>Banana</ActionListItem>
+          <ActionListItem selected={colors.includes('Orange')}>Orange</ActionListItem>
+          <ActionListItem selected={colors.includes('Pineapple')}>Pineapple</ActionListItem>
+        </ActionListGroup>
+      </ActionList>
+    </ActionMenu>
+  )
+}
+```
+
+### Links
+
+`ActionMenu` can also display links within its action list to navigate to the specified url when that action is activated:
+
+```jsx example
+<ActionMenu
+  trigger={
+    <Button variant="secondary">
+      Actions with Links
+      <Icon type="chevron-down" />
+    </Button>
+  }
+>
+  <ActionList>
+    <ActionListLinkItem href="#account">My Account</ActionListLinkItem>
+    <ActionListLinkItem href="#settings">Settings</ActionListLinkItem>
+    <ActionListSeparator />
+    <ActionListItem>Logout</ActionListItem>
+  </ActionList>
+</ActionMenu>
+```
+
+## Props
+
+### ActionMenu
+
+<ComponentProps
+  props={[
+    {
+      name: 'children',
+      type: 'React.ReactElement',
+      required: true,
+      description: 'The ActionList component to be shown when the menu is opened.'
+    },
+    {
+      name: 'trigger',
+      type: ['React.ReactElement', '(props: ActionMenuTriggerProps, open: boolean) => React.ReactElement'],
+      required: true,
+      description: 'Element that triggers the menu to open. Can be a React element or a render function that receives props and the current open state.'
+    },
+    {
+      name: 'placement',
+      type: 'string',
+      defaultValue: 'bottom-start',
+      description: 'Positional placement value to anchor the overlay element relative to its anchored target.'
+    },
+    {
+      name: 'open',
+      type: 'boolean',
+      description: 'Controls whether the action menu is open. Only necessary for controlling action menu state.'
+    }
+  ]}
+/>
+
+## Related Components
+
+- [ActionList](./ActionList)
+- [AnchoredOverlay](./AnchoredOverlay)
```

Comment:
Should we add `ActionMenu` to the "Related Components" section of `AnchoredOverlay`? I think we're listing components that also use `AnchoredOverlay` there like `Tooltip` and `Popover`.

Reply:
I'm not sure, I don't have strong opinions here. Related components is typically components you would want to use together (or components that need referencing). How strongly do you feel about this? I'm probably good either way.